modules/nw-about-multicast.adoc

```diff
@@ -1,12 +1,7 @@
 // Module included in the following assemblies:
 //
-// * networking/openshift_sdn/enabling-multicast.adoc
 // * networking/ovn_kubernetes_network_provider/enabling-multicast.adoc
 
-ifeval::["{context}" == "openshift-sdn-enabling-multicast"]
```

Comment:
We could ultimately refactor out the conditionals in every instance in every file; I don't know if that's in scope, though. It certainly doesn't hurt to leave them.

Reply:
Minor point, but make sure to remove the corresponding ifeval block:
```
ifeval::["{context}" == "openshift-sdn-enabling-multicast"]
:!openshift-sdn:
endif::[]
```

Reply:
Yeah, we never ever want to try to support two network plugins at the same time again in the future, so that sort of conditionalization is not likely to be useful again.

Reply:
Gone.